src/config.rs

```diff
@@ -452,6 +459,27 @@ fn parse_dependency(name: impl Into<String>, value: &Item) -> Result<Dependency>
         return Ok(
             HttpDependency { name, version: version.to_string(), url: None, checksum: None }.into()
         );
+    } else if value.is_inline_table() && // TODO: Hacky way of doing this, might need rewritten
+        !value.as_inline_table().unwrap().contains_key("url") &&
+        !value.as_inline_table().unwrap().contains_key("git")
+    {
+        // this function does not retrieve the url, only version
+        return Ok(HttpDependency {
+            name: name.clone(),
+            version: match value.as_inline_table() {
+                // we normalize to inline table
+                Some(table) => {
+                    let version = table.get("version").unwrap().to_string();
+                    version.replace("\"", "").trim().to_string()
+                }
+                None => {
+                    return Err(ConfigError::InvalidDependency(name));
+                }
+            },
+            url: None,
+            checksum: None,
+        }
+        .into());
```

Comment:
You can remove this and instead handle this case in the `None` case of the last match statement